Dockerfile.assisted-swarm

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/centos/centos:stream8
+
+ARG WORK_DIR=/data
+
+RUN mkdir $WORK_DIR && chmod 775 $WORK_DIR
+COPY --from=quay.io/edge-infrastructure/assisted-installer:latest /assisted-installer-controller/deploy /assisted-installer-controller/deploy
```

Comment:
What do we need this for?

Reply:
https://github.com/openshift/assisted-installer/blob/912c3b26e4f1425bd362b416c90a69e1a9822089/src/ops/ops.go#L65

Reply:
https://github.com/openshift/assisted-installer/blob/912c3b26e4f1425bd362b416c90a69e1a9822089/src/ops/ops.go#L395

Reply:
Ah I see. I think it's better we skip `PrepareController` altogether in installer dry run mode. Then we could get rid of having the installer annoyingly depend on the existence of these files.

I'm also afraid they may change in the future and we'd have potential incompatibility between the swarm go.mod installer dependency and the quay :latest installer image

Reply:
Will you take care of that? Or should I?

Reply:
https://github.com/openshift/assisted-installer/pull/447